measure_cocoapod_size.py

```diff
@@ -214,6 +214,18 @@ def GetPodSizeImpact(parsed_args):
     podfile_dict['combined_pods_extra_size'] = target_size - source_size
     with open(parsed_args.json, 'w') as json_file:
       json.dump(podfile_dict, json_file)
+  # Throw an error if the target size is 0, an example for command
+  # ./measure_cocoapod_size.py --cocoapods FirebaseABTesting FirebaseAut:8.0.0
+  # This command will throw the following error:
```

Comment:
Why does this command throw an error? Because `FirebaseAut` is misspelled?

Reply:
Yes, will update the podname in the comment to make it more clear as an error.